deploy/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -3293,12 +3291,17 @@ spec:
                 - -https-address=:8888
                 - -http-address=
                 - -email-domain=*
-                - -upstream=https://localhost:8080/onboarding-tokens
+                - -upstream=https://ux-backend-proxy.openshift-storage.svc:8080/onboarding-tokens
```

Comment:
seems it was missed in earlier reviews, proxy for a single path in the api,like `onboarding-tokens` is a big no.